code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1633,6 +1629,18 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
 		H.receiving_cpr = FALSE
 		to_chat(src, "<span class='danger'>You need to stay still while performing CPR!</span>")
 
+/mob/living/carbon/human/proc/get_cpr_mod(mob/living/carbon/human/H)
+	if((head && (head.flags_cover & HEADCOVERSMOUTH)) || (wear_mask && (wear_mask.flags_cover & MASKCOVERSMOUTH) && !wear_mask.mask_adjusted))
+		return CPR_CHEST_COMPRESSION_ONLY
+	if((H.head && (H.head.flags_cover & HEADCOVERSMOUTH)) || (H.wear_mask && (H.wear_mask.flags_cover & MASKCOVERSMOUTH) && !H.wear_mask.mask_adjusted))
+		return CPR_CHEST_COMPRESSION_ONLY
+	if(!H.check_has_mouth() || !check_has_mouth())
+		return CPR_CHEST_COMPRESSION_ONLY
```

Comment:
(Probably) unintended side effect of this is that IPCs can be given chest compressions now when they're in crit. It doesn't actually *do* anything since they don't take oxyloss damage, but it maybe shouldn't be possible to do in the first place.